src/fourierdomain/nuft/nfft_alg.jl

```diff
@@ -107,6 +107,28 @@ end
     return nothing
 end
 
+function EnzymeRules.forward(
+    config::EnzymeRules.FwdConfig,
+    func::Const{typeof(_jlnuft!)}, 
+    ::Type{RT}, 
+    out::Annotation{<:AbstractArray{<:Complex}}, A::Const{<:NFFTPlan},
+    b::Annotation{<:AbstractArray{<:Real}}
+) where RT
+    # Forward rule does not have to return any primal or shadow since the original function returned nothing
+    func.val(out.dval, A, b.dval)
```

Comment:
```suggestion
    func.val(out.dval, A.val, b.dval)
```
You need to extract the plan from `A`

Reply:
Done